README.md

```diff
@@ -121,57 +127,13 @@ For details on environment configuration, including optional variables, see
 
 ## Running the Server
 
-The easiest way to develop MBTA dotcom is to use Docker Compose.
```

Comment:
We still want to keep these instructions because some things can only be tested with the docker compose setup. It is the most similar to our production environment. E.g., I send a command to server 1 to delete something from the cache and I want to make sure it has been deleted from server 2 as well.

Reply:
I'll move this content to the new file as well.